apps/node-host/webpack.config.js

```diff
@@ -8,13 +8,19 @@ const { UniversalFederationPlugin } = require('@module-federation/node');
 module.exports = composePlugins(withNx(), (config) => {
   // Update the webpack config as needed here.
   // e.g. `config.plugins.push(new MyPlugin())`
+  config.cache = false;
   delete config.module.rules[config.module.rules.length - 1];
   config.plugins.push(
     new UniversalFederationPlugin({
       isServer: true,
       name: 'node_host',
       remotes: {
-        node_remote: 'node_remote@http://localhost:3002/remoteEntry.js',
+        node_remote:
+          'globalThis.__remote_scope__.cache.node_remote@http://localhost:3002/remoteEntry.js',
```

Comment:
This needs improvement. Somewhere in webpack we should make new externalType like "http-vm global@url". Like promise new promise